src/MetaInfo/Form/DefaultType.php

```diff
@@ -1,9 +1,11 @@
 <?php
 
+namespace Redaxo\Core\MetaInfo\Form;
+
 /**
  * @api
  */
-final class rex_metainfo_default_type
+final class DefaultType
```

Comment:
Bei dem `Form`-Namespace bin ich gestolpert. Die Klasse dient ja vor allem dazu, den Default-Metainfo-Typen feste IDs in der DB zuzuordnen. Also könnte auch in `Database` gehören. 

Aber andererseits ist es auch ein Enum allgemein über die Default-Feldtypen, die für die Metainfos bereitstehen. So gesehen passt es doch auch in `Form`.

Man könnte auch überlegen, es eins hochzuschieben, allgemein in den `MetaInfo`-Namespace, da übergreifend über die Subnamespaces.

Für alle drei Varianten spricht was, und wäre mit allen einverstanden denke ich. Kann somit in `Form` bleiben. Aber wollte meine Gedanken dazu mal teilen ;) 

(Können übrigens auch überlegen, ein Enum draus zu machen. Aber lieber nicht hier im PR)

Reply:
Gern ein Enum in einem separaten PR.